src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        DateTimeFormatter fomater = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate localeFrom = LocalDate.parse(dateFrom, fomater);
+        LocalDate localeTo = LocalDate.parse(dateTo, fomater);
+        int salary = 0;
+        StringBuilder report = new StringBuilder(" Report for period "
+                + dateFrom + " - "
+                + dateTo + System.lineSeparator());
+
+        if (localeFrom.isAfter(localeTo)) {
+            throw new RuntimeException("Invalid dates");
+        }
```

Comment:
```suggestion
        if (localeFrom.isAfter(localeTo)) {
            throw new RuntimeException("Invalid dates. Date from: " + localeFrom + " could'd be after  date to: " localeTo);
        }
```

lets make exception messages more informative